client/components/buttons/iconButtons/deleteButton.tsx

```diff
@@ -22,7 +22,7 @@ export default function DeleteButton({
 
     const handleClick = () => {
         axios
-            .delete(`${backend_uri}/pool/content/${poolItem.uri}`, {
+            .delete(`${backend_uri}/pool/content/${poolItem.spotify_icon_uri}`, {
```

Comment:
This seems wrong. The item uri is the one that should go to backend, not the icon uri

Reply:
There is an issue where is should either send spotify_collection_uri or spotify_track_uri depending on whats being deleted
will look into this today

Reply:
This has been resolved